include/mppic/controller.hpp

```diff
@@ -52,6 +52,7 @@ class Controller : public nav2_core::Controller
   TrajectoryVisualizer trajectory_visualizer_;
 
   bool visualize_;
+  double base_x_vel_, base_y_vel_, base_theta_vel_;
```

Comment:
This is another case of mirroring other `Controller` plugins' implementations. This differs because this is kept static and known while the max_vx_ max_vy_ may be dynamically adjusted at runtime through the Speed Limiting feature of Nav2's controller server. 

Over time those can be changed and we need to know what the "baseline" maximum speeds were set to in order to preform requests like "lower speed to 50% of maximum" then "lower speed to 10% of maximum". For consecutive changes, we need to know what the baseline values are, which we name `base_` as a matter of convention in the other plugins, to differentiate between the current maximum speed and the baseline maximum speed should we want to reset or a new request comes in that we need to interpret. 

Otherwise, we'd be lowering the 50%-reduced speed another 10%, rather than working with an absolute baseline. We need to store it somewhere.